language/stdlib/modules/LibraAccount.move

```diff
@@ -472,6 +476,11 @@ module LibraAccount {
         );
         withdraw_from_balance<Token>(payer, payee, account_balance, amount)
     }
+    spec fun withdraw_from {
+        ensures forall addr: address where old(exists<Balance<Token>>(addr)) && addr != cap.account_address:
+            global<Balance<Token>>(addr).coin.value == old(global<Balance<Token>>(addr).coin.value); // This function can only withdraw from the balances of cap.account_address [B27].
+        // TODO: this spec block is incomplete.
```

Comment:
Could you tag with your unixname for later grepping?

Reply:
Done.